package.json

```diff
@@ -36,29 +36,32 @@
     "walletconnect"
   ],
   "dependencies": {
-    "@ethersproject/sha2": "5.7.0",
-    "@solana/web3.js": "1.63.0",
-    "@walletconnect/qrcode-modal": "1.8.0",
-    "@walletconnect/universal-provider": "^2.1.0",
+    "@ethersproject/sha2": "^5.7.0",
+    "@walletconnect/universal-provider": "^2.1.3",
     "bn.js": "5.2.1",
     "borsh": "0.7.0",
     "bs58": "5.0.0",
+    "buffer": "^6.0.3",
     "valtio": "1.7.0"
   },
   "peerDependencies": {
-    "buffer": "6.0.3"
+    "@solana/web3.js": "1.63.0",
+    "@walletconnect/qrcode-modal": "1.8.0"
   },
   "devDependencies": {
+    "@esbuild-plugins/node-globals-polyfill": "^0.1.1",
+    "@solana/web3.js": "1.63.0",
     "@types/bn.js": "5.1.1",
     "@typescript-eslint/eslint-plugin": "5.40.0",
     "@typescript-eslint/parser": "5.40.0",
+    "@walletconnect/qrcode-modal": "1.8.0",
     "better-sqlite3": "7.6.2",
```

Comment:
Why is better-sqlite3 needed? I think Gancho swaped it out to more lightweight package that is pre-bundled into v2?

Reply:
Sometimes, when running example-next with the older versions of universal provider this was needed. I'll double check